calicoctl/commands/node/status.go

```diff
@@ -48,12 +44,27 @@ calicoctl node status
 Description:
   Display the status of the Calico node`
 
+	// Note: This call is ignoring the error because error check happens at the level above
+	// i.e at `node.go` before it calls `node.Status`. This call is just so help message gets
+	// printed for this option
 	_, _ = docopt.Parse(doc, args, true, "calicoctl", false, false)
 
 	ctx := context.Background()
 
-	defaultHeaders := map[string]string{"User-Agent": "engine-api-cli-1.0"}
-	cli, err := client.NewClient("unix:///var/run/docker.sock", "v1.22", nil, defaultHeaders)
+	if os.Getenv("DOCKER_HOST") == "" {
+		err := os.Setenv("DOCKER_HOST", "unix:///var/run/docker.sock")
+		if err != nil {
+			log.Fatalf("Error setting DOCKER_HOST: %v", err)
+		}
+	}
+	err := os.Setenv("DOCKER_API_VERSION", "1.16")
```

Comment:
added a check to see if it's set, if it is then don't set it, else set it to 1.16 (like the python version